localization/english/religion/wc_religion_l_english.yml

```
@@ -5,4 +5,502 @@
+ 
+ forsaken_cult:0 "Forgotten Shadow"
+ forsaken_cult_adj:0 "Forgotten"
+ forsaken_cult_adherent:0 "Forgotten"
+ forsaken_cult_adherent_plural:0 "Forgotten"
+ forsaken_cult_desc:0 "A heretical group of shadow practitioners, the Forgotten Shadow, believe that dark energy practitioners are just as vital to the survival of life and reality as wielders of the Light. The cult preaches balance, mandating that shadow priests must never forget the dangers being corrupted by the dark energies they manipulate, for it is often difficult to find one's way back to the shadow from the deep reaches of the darkness."
+ 
+ water_group:0 "Hydromantism"
+ water_group_adj:0 "Hydromantic"
+ water_group_adherent:0 "Hydromantist"
+ water_group_adherent_plural:0 "Hydromantists"
+ water_group_desc:0 "Among all the elements, $water_group_adherent_plural$ regard water as the main force and worship water elementals as deities."
+ 
+ neptulon_worship:0 "Neptulism"
+ neptulon_worship_adj:0 "Neptulic"
+ neptulon_worship_adherent:0 "Neptulist"
+ neptulon_worship_adherent_plural:0 "Neptulists"
+ neptulon_worship_desc:0 "$fire_cult_adherent_plural$ worship one of the Elemental Lords, Neptulon the Tidehunter, as their main god. Other fire elementals are seen as minor deities."
+ 
+ waterspeaker:0 "Waterspeaking"
+ waterspeaker_adj:0 "Waterspeaker"
+ waterspeaker_adherent:0 "Waterspeaker"
+ waterspeaker_adherent_plural:0 "Waterspeakers"
+ waterspeaker_desc:0 "Waterspeaking is an ancient belief that originated in Pandaria before the Great Sundering, that allows its followers to commune with the land through rituals involving riverwater. The ones to maintain the long tradition of waterspeaking long after the mists have concealed the continent were the fish-people known as jinyu. Over time the wisest of their people learned to commune with the waters of the river and listen for the future and news it would bring. One of the point of scholarly debates is the Waterspeaker magic's unclear relation to the larger school of hydromancy, but many rumor it to be closely related yet evolved into a major offshoot due to centuries of isolation."
+ 
+ water_deities:0 "Mrligion"
+ water_deities_adj:0 "Mrligions"
+ water_deities_adherent:0 "Mrhermnt"
+ water_deities_adherent_plural:0 "Mrhermnts"
+ water_deities_desc:0 "The ones who believe in water deities follow rituals involving communication with the sea and its powers, and they worship the water and the powerful entities within it. The pantheon of the water deities is dynamic, changing to include new creatures they encounter. Even a strong shark or a quick fish may become a living symbol of the faith. As the animals die or move on, the murloc faith mutates to find new symbolism in their surroundings."
```

Changed the first sentence.
```suggestion
 water_deities_desc:0 "Those who believe in water deities follow rituals involving communication with the sea and its powers, and they worship the water and any powerful entities within it. The pantheon of the water deities is dynamic, changing to include new creatures they encounter. Even a strong shark or a quick fish may become a living symbol of the faith. As the animals die or move on, the murloc faith mutates to find new symbolism in their surroundings."
```